ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.js

```diff
@@ -58,4 +67,14 @@ export default class MessagesList extends Component {
       this.invalidFormAlert = 'There was an error submitting this form.';
     }
   }
+
+  @action
+  updateMultipleMessageModal() {
+    const { isValid, state, invalidFormMessage } = this.args.message.validate();
```

Comment:
why does this validation happen here and in the save task?

Reply:
@hellobontempo Good question! Open to thoughts on this! My thought was to check for validations before the modal message pops up instead of having the user go through the whole modal flow just to find out they needed to add a title and/or message.

![custom-messages-mulitple](https://github.com/hashicorp/vault/assets/30884335/1b8cb8e2-fa95-43fc-a235-ebd01f9c3a70)

Reply:
I just updated the code so it won't validate twice!

Reply:
It feels like slightly confusing UX, though I understand the intent here. So perhaps a question for design! My thoughts would be that validation should happen prior to save, as usual. But if a user has confirmed they intend to create a second modal message - they shouldn't have to click "Confirm" again. So rather clicking "confirm" in the modal saves the user choice rather than submits the form. But it might be good to see what other folks think!

Reply:
sounds good!